W-17564477 add docs for custom image names (#761)
mulesoft/docs-gateway: pdk/1.3/modules/ROOT/pages/policies-pdk-integration-tests.adoc

NOTE: If the Docker engine is properly configured, this test passes. Run `make test` to ensure proper configuration.

[[alternative-httpmock-docker-images]]
=== Configure an Alternative httpmock Docker Image

To use an `httpmock` Docker image from another source, instead of the provided default, the configuration builder provides an `image_name()` method to define it explicitly. 

> To configure an alternative `httpmock` Docker image, update your your `HttpMockConfig::builder()` definition as in the following example:

Reviewer: ```suggestion
To configure an alternative `httpmock` Docker image, update your your `HttpMockConfig::builder()` definition as in this example:
```

Reviewer: This is an editorial thing, I'm really bad about using "the following".

}
----

[[alternative-flex-docker-images]]
=== Configure an Alternative Flex Docker Image

To use a `flex` Docker image from another source, instead of the provided default, the configuration builder provides an `image_name()` method to define it explicitly. 

> To configure an alternative Flex Docker image, update your your `FlexConfig::builder()` definition as in the following example:

Reviewer: ```suggestion
To configure an alternative Flex Docker image, update your your `FlexConfig::builder()` definition as in this example:
```
